### ISSUE TITLE
Typo in help section 

### ISSUE THREAD

**Issue by IsmailM** [heathsc/gemBS#11]
https://github.com/heathsc/gemBS/blob/6dc10849e5594754e1f28834b1d3975e1c77c3e7/src/production.py#L386

Please update to something like: `Path where the BAM aligned files are located` (using where instead of were)

i.e. 

```
 parser.add_argument('-i', '--input-dir', dest="input_dir",metavar="PATH", help='Path where the BAM aligned files are located.', required=True) 
```

**Comment by MarcosFernandez** [comment]
Hi Ismail,

Thanks a lot for reporting this issue. It will be fixed in coming version.

Cheers,
Marcos.

**Comment by MarcosFernandez** [comment]
The typo has been fixed on the recently published 2.0 version.

Again thanks,
Marcos.
